packages/react/src/components/login/client/oidcClient.ts

```diff
@@ -0,0 +1,301 @@
+import { UserManager, UserManagerSettings, WebStorageStateStore, User, Log } from 'oidc-client-ts';
+import to from 'await-to-js';
+import jwtDecode from 'jwt-decode';
```

Comment:
`'jwt-decode' should be listed in the project's dependencies. Run 'npm i -S jwt-decode' to add iteslint[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.28.0/docs/rules/no-extraneous-dependencies.md)`

Reply:
This is weird. The package is listed in package.json and this error does not show in my local env.  And the project is built here just fine. Have you ran yarn install when switching branches? Your IDE seems to warn about all new packages, so that might be it.

Reply:
Seems still to work here. Build and tests pass.